src/library/Hooks/useInflation/index.tsx

```diff
@@ -39,7 +39,7 @@ const useYearlyInflation = () => {
 
   useEffect(() => {
     getYearlyInflation?.()
-      .then((val) => setYearlyInflation(val.toNumber() / 1_000_000_000))
+      .then((val) => setYearlyInflation(Number(val) / 1_000_000_000))
```

Comment:
Did you check if this works? I don't think I've seen conversion made this way before :thinking:

Reply:
Unfortunately, the val of type Codec doesn't have a .toNumber(), so I used the default conversion

`const x = Number("1")
const y = Number("ala")
const z = Number(undefined)

console.log(x / 1000000000)
  1e-9
console.log(x / 1_000_000_000)
  1e-9
console.log(y / 1_000_000_000)
  NaN
console.log(z / 1_000_000_000)
  NaN
`